### ISSUE TITLE
Default migration prefix same as scout

### ISSUE THREAD

**Issue by Surt** [babenkoivan/elastic-migrations#15]
https://github.com/babenkoivan/elastic-migrations/blob/2a1998ad9fd1934bb161e87f675efcc5e7efb6bf/config/elastic.migrations.php#L6

I did published the config and changed it by hand but, it seems like a better default to use the scout prefix? So the migrations and the scout doesn't use different values by a wrong config.

```php
    'index_name_prefix' => env('ELASTIC_MIGRATIONS_INDEX_NAME_PREFIX', env('SCOUT_PREFIX', '')),
```

**Comment by babenkoivan** [comment]
I didn't use `SCOUT_PREFIX` here, cause Elastic Migrations can be used as a standalone package, i.e. without Scout. 
I'll give it another thought, maybe it makes sense.

**Comment by babenkoivan** [comment]
Hey @Surt, it's now implemented in [v1.3.1](https://github.com/babenkoivan/elastic-migrations/releases/tag/v1.3.1) 🙂
